Hearthstone Deck Tracker/Controls/Overlay/BattlegroundsMinions.xaml.cs

```diff
@@ -91,8 +101,8 @@ private void Update(int tier, IEnumerable<Race> availableRaces)
 					Groups.FirstOrDefault(x => x.Title == title)?.Hide();
 				else
 				{
-					var available = race == Race.ALL || race == Race.INVALID || availableRaces.Contains(race);
-					resort |= AddOrUpdateBgCardGroup(title, cards, available);
+					if(race == Race.ALL || race == Race.INVALID || availableRaces.Contains(race))
+						resort |= AddOrUpdateBgCardGroup(title, cards, true);
```

Comment:
I think `AddOrUpdateBgCardGroup` is only ever called for available types now? So you can probably get rid of the 3rd parameter.